lib/views/commit-view.js

```diff
@@ -63,6 +93,17 @@ export default class CommitView extends React.Component {
       this.props.commandRegistry.add('atom-workspace', {
         'github:commit': this.commit,
         'github:toggle-expanded-commit-message-editor': this.toggleExpandedCommitMessageEditor,
+
+        'github:co-author:down': this.proxyKeyCode(40),
+        'github:co-author:up': this.proxyKeyCode(38),
+        'github:co-author:enter': this.proxyKeyCode(13),
+        'github:co-author:tab': this.proxyKeyCode(9),
+        'github:co-author:backspace': this.proxyKeyCode(8),
+        'github:co-author:pageup': this.proxyKeyCode(33),
+        'github:co-author:pagedown': this.proxyKeyCode(34),
+        'github:co-author:end': this.proxyKeyCode(35),
+        'github:co-author:home': this.proxyKeyCode(36),
+        'github:co-author:delete': this.proxyKeyCode(46),
```

Comment:
Random sidenote: we could move these into a [`<Commands>`](https://github.com/atom/github/blob/master/lib/views/commands.js) view for extra React Points.